tests/Console/Command/FixCommandTest.php

```diff
@@ -56,6 +57,25 @@ public function testEmptyFormatValue()
         static::assertSame(0, $cmdTester->getStatusCode(), "Expected exit code mismatch. Output:\n".$cmdTester->getDisplay());
     }
 
+    public function testMultipleFilesWithoutConfigFile()
+    {
+        $cmdTester = $this->doTestExecute(
+            [
+                'path' => [
+                    __FILE__,
+                    __DIR__.'/DescribeCommandTest.php',
+                ],
+                '--rules' => 'switch_case_semicolon_to_colon',
+            ]
+        );
+
+        $output = $cmdTester->getDisplay(true);
+
+        static::assertNotFalse(strpos($output, 'Loaded config default.'));
```

Comment:
it's not obvious why loading 2 unrelated things (fixing multiple files + passing rules by cli param) is making tool to ignore the config file.

`php-cs-fixer fix src/path1 src/path2 --rules=foo` and boom, whole `.php_cs.dist` got ignored

Reply:
Are you claiming `php-cs-fixer fix src/path1 src/path2 --rules=foo` runs currently? Cause it does not.

Reply:
i'm not claiming. i'm saying.
after change proposed with this PR, running given command would ignore configuration file totally, which is wrong.

Reply:
> running given command would ignore configuration file totally, which is wrong.

given your specs, I think it is totally fine

Reply:
if we ignore the config, we shall at least give user the note about it.
i don't want any implicit actions simply.

maybe let's have the `--config=default` or `--no-config` to allow for such cases?

Reply:
`Passing both `config` and `rules` options is not possible. `

It is already implicit which config file will be used when you use `rules` because you are not allowed to pass one explicit. So making it explicit the default config with the commandline arguments override seems best to me.

Reply:
yeah, i guess it was a preparation for killing --rules in general